packages/edit-site/src/components/sidebar-navigation-screen-pattern/use-pattern-details.js

```diff
@@ -80,11 +86,11 @@ export default function usePatternDetails( postType, postId ) {
 
 	const details = [];
 
-	if ( postType === 'wp_block' || 'wp_template_part' ) {
+	if ( postType === PATTERN_TYPES.user || TEMPLATE_PART_POST_TYPE ) {
```

Comment:
Wait, I just found out that this condition doesn't make sense at all 😂. 

Shouldn't it be:

```suggestion
	if ( postType === PATTERN_TYPES.user || postType === TEMPLATE_PART_POST_TYPE ) {
```

? 😅

Reply:
Oh, good spotting. Looks like a safe change for this PR. 

Thanks!

Reply:
Just needed a lint :D